00-open-a-shapefile.Rmd

```diff
@@ -8,9 +8,9 @@ contributors: [Sarah Newman]
 dateCreated:  2015-10-23
 lastModified: `r format(Sys.time(), "%Y-%m-%d")`
 packagesLibraries: [rgdal, raster]
-category: [self-paced-tutorial]
+category: 
 mainTag: vector-data-workshop
-tags: [vector-data, R, spatial-data-gis]
+tags: [vector-data, vector-data-workshop, R, spatial-data-gis]
```

Comment:
This was removed intentionally 

https://github.com/data-lessons/NEON-R-Spatial-Vector/issues/72

i was using this repo to test. i think we need new code. We have the 
workshopSeries: [vector-data-series]
YAML element now which means we don't need a workshop tag - it's redundant. However the code for the bottom of the page renders using the tags and mainTags. the fix will be some tricky coding i think.